app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -360,20 +358,21 @@ public class MainActivity : AppCompatActivity(), ViewController, Router.Callback
         optionsMenu?.findItem(R.id.action_search)?.collapseActionView()
     }
 
-    override fun showRoutePreview(feature: Feature) {
+    override fun showRoutePreview(feature: Feature?) {
```

Comment:
Why does this have to be nullable? If there is no way around changing the param type we should probably null check `feature` before we proceeding.